code/modules/antagonists/valentines/valentine.dm

```diff
@@ -2,6 +2,7 @@
 	name = "valentine"
 	roundend_category = "valentines" //there's going to be a ton of them so put them in separate category
 	show_in_antagpanel = FALSE
+	no_mulligan = TRUE
```

Comment:
Isn't this the precise opposite

Reply:
lol.

The var name is kinda confusing, it should be `can_delay_round` or something

Reply:
Looks like a logic error as well, unless I'm just being dumb.
no_mulligan being true here -> var/no_mulligan is true -> return  0

Reply:
sounds "right"

Reply:
no mulligan turned out better than mulligan_skip trust me, i like the default on this being FALSE

Reply:
i'll switch it to prevent_mulligan, seems a lot clearer